### ISSUE TITLE
UIWindow's SendMessage miss

### ISSUE THREAD

**Issue by delgelato** [vis2k/uMMORPG_CE#37]
UIWindow has OnWindowDrag and OnWindowClose, gets sent to UIWindow.window

UIWindow.window is the moveable gameobject, but not the GO that usually contains the UISkills, UIInventory etc. That's 1 more parent up

So either do the SendMessage to window.parent, or scrap that and use an Action instead, and UISkills etc can listen to that
This means UISkills and all the other UI menu scripts will need to have reference of the UIWindow or GetComponentInChildren... but i personally dont like SendMessage so this is what i already did anyways

Slightly unrelated:
Eventually you'll move to UI Toolkit, which is what i tried bcoz i also can't wait. But some APIs are still missing/some dont work yet. I gave up [here on the last post](https://forum.unity.com/threads/how-to-detect-that-a-textfield-is-focused.966491/) when i can't manage to Unfocus a TextField..
So in short, i dont think UITK is ready for rpg kinda menu. It's only good enough for permanent, non interactable menus in runtime for now

**Comment by miwarnec** [comment]
yep UnityEvent seems better.
will take a look soon.
